src/test/java/org/takes/facets/auth/codecs/CcStrictTest.java

```diff
@@ -67,5 +67,12 @@ public void passesValid() throws IOException {
                 )
             ), Matchers.equalTo("urn%3Atest%3A1")
         );
+        MatcherAssert.assertThat(
+            new String(
+                new CcStrict(new CcPlain()).encode(
+                    new Identity.Simple("urn:test:1:1")
```

Comment:
@dmzaytsev let's test with the urn provided in the original ticker: `urn:test-domain-org:valid:1`